packages/server/graphql/private/mutations/toggleAllowInsights.ts

```diff
@@ -0,0 +1,39 @@
+import {r} from 'rethinkdb-ts'
+import {getUsersByEmails} from '../../../postgres/queries/getUsersByEmails'
+import {MutationResolvers} from '../resolverTypes'
+
+const toggleAllowInsights: MutationResolvers['toggleAllowInsights'] = async (
+  _source,
+  {value, domain, emails},
+  {dataLoader}
+) => {
+  const organizations = await dataLoader.get('organizationsByActiveDomain').load(domain)
+  if (organizations.length === 0) {
+    return {
+      error: {message: `No organizations match ${domain}. First call 'setOrganizationDomain'`}
+    }
+  }
+  const orgIds = organizations.map(({id}) => id)
+  if (emails.length < 1) {
+    return {error: {message: 'Must include at least 1 email'}}
+  }
+  const users = await getUsersByEmails(emails)
```

Comment:
-1 if there aren't any users, e.g. if there's a typo in the email provided, `recordsReplaced` will be an empty array and we'll throw an error on line 36: `Reduce of empty array with no initial value`. So, I think we should check if `users` length is 0 and return an error if it is